src/txn/txn_timestamp.c

```diff
@@ -553,14 +567,24 @@ __wt_timestamp_validate(WT_SESSION_IMPL *session, const char *name,
 	    txn_global->has_stable_timestamp &&
 	    __wt_timestamp_cmp(ts, &txn_global->stable_timestamp) < 0));
 
-	if (older_than_oldest_ts)
+	if (older_than_oldest_ts) {
+		WT_WITH_TIMESTAMP_READLOCK(session, &txn_global->rwlock,
+		    ret = __wt_timestamp_to_hex_string(
+		    session, hex_timestamp, &txn_global->oldest_timestamp));
```

Comment:
Here, it is possible that txn_global->oldest_timestamp gets updated by another thread. So it would be different from from what it might have been originally at line#565. If this happens the error message will show an incorrect value of the oldest timestamp. 
Can we save the oldest_timestamp in line#565 into a local variable while we have the lock and use that saved value to display in the error message?